mesh/notifications/models.py

```diff
@@ -1,7 +1,11 @@
 from django.db import models
+from mesh.accounts.models import Account
 
 # Create your models here.
 
 class Notification(models.Model):
-    #TODO
-    pass
\ No newline at end of file
+    accountID = models.ForeignKey(Account, on_delete=models.CASCADE)
+    notificationID = models.AutoField(primary_key=True)
+    timeStamp = models.TimeField()
+    notificationContextJSON = models.JSONField()
+    
```

Comment:
Should this table also have a Meta class for metadata or is it unnecessary, otherwise looks good to me!